packages/server/src/auth/external.ts

```diff
@@ -187,10 +187,16 @@ async function verifyExternalCode(
       body: params.toString(),
     });
 
-    const tokens = await response.json();
-    return parseJWTPayload(tokens.id_token);
+    const responseBody = await response.json();
+
+    if (!response.ok) {
+      globalLogger.warn('Bad response from external auth check', { status: response.status, body: responseBody });
```

Comment:
@codyebberson  Related to my comment above, I think `body` should be set to `await response.text()` here.